libappstream-glib/as-enums.h

```diff
@@ -84,6 +85,7 @@ typedef enum {
 	AS_URL_KIND_HELP,		/* Since: 0.1.5 */
 	AS_URL_KIND_MISSING,		/* Since: 0.2.2 */
 	AS_URL_KIND_TRANSLATE,		/* Since: 0.6.1 */
+	AS_URL_KIND_DETAILS,		/* Since: 0.7.15 */
```

Comment:
In libappstream I created a `AS_RELEASE_URL_KIND` to distinguish between what URLs are valid in releases and which ones aren't.
That distinction is probably not necessary though.